### PR TITLE
chore: remove bump script from package.json

### PR DESCRIPTION
**Pull request by SunMoon97**
## The basics

- [x] I [validated my changes](https://developers.google.com/blockly/guides/contribute/core#making_and_verifying_a_change)

### Resolves

Fixes https://github.com/google/blockly/issues/7543

### CODE REVIEW

**Comment by BeksOmega** [comment]
Heya @SunMoon97 Thanks for submitting this fix :D Could you:
1) Fill out the PR template.
2) [Fix your commit message](https://developers.google.com/blockly/guides/contribute/get-started/commits#fixing_non-conventional_commits). It should be `chore: remove bump script from package.json`. We automatically generate release notes from our commit messages, so they need to be specially formatted =)

After that this should be good to go!

**Comment by SunMoon97** [comment]
@BeksOmega I did as instructed .Please see to it.

File: package.json [hunk 1]
```diff
@@ -98,7 +97,7 @@
     "markdown-tables-to-json": "^0.1.7",
     "mocha": "^10.0.0",
     "patch-package": "^8.0.0",
-    "prettier": "3.0.3",
+    "prettier": "^3.0.3",
```
Comment (BeksOmega) [inline package.json:100]: Heya @SunMoon97, could you revert this change + the change to the package-lock? Prettier recommends using the exact version so that you make sure everyone is always formatting the files the same way!

Comment (SunMoon97) [reply package.json]: Done

**Review by BeksOmega — APPROVED** [review]
This looks great! Thank you for your hard work on this!
